src/object.d

```diff
@@ -1627,6 +1629,16 @@ class Throwable : Object
             }
         }
     }
+
+    /**
+     * Get the message describing the error.
+     * Returns:
+     *  message
```

Comment:
Document here that by default `message()` returns the `msg` member, and if the user prefers to build the error message a different way they may want to override this method.